libs/shared/contentful/codegen.config.ts

```diff
@@ -1,12 +1,14 @@
 import type { CodegenConfig } from '@graphql-codegen/cli';
 
 const CONTENTFUL_GRAPHQL_API_URL = process.env.CONTENTFUL_GRAPHQL_API_URL;
-
 const CONTENTFUL_GRAPHQL_API_KEY = process.env.CONTENTFUL_GRAPHQL_API_KEY;
+const CONTENTFUL_GRAPHQL_SPACE_ID = process.env.CONTENTFUL_GRAPHQL_SPACE_ID;
+const CONTENTFUL_GRAPHQL_ENVIRONMENT =
+  process.env.CONTENTFUL_GRAPHQL_ENVIRONMENT;
 
 const config: CodegenConfig = {
   overwrite: true,
-  schema: `${CONTENTFUL_GRAPHQL_API_URL}?access_token=${CONTENTFUL_GRAPHQL_API_KEY}`,
+  schema: `${CONTENTFUL_GRAPHQL_API_URL}/spaces/${CONTENTFUL_GRAPHQL_SPACE_ID}/environments/${CONTENTFUL_GRAPHQL_ENVIRONMENT}?access_token=${CONTENTFUL_GRAPHQL_API_KEY}`,
```

Comment:
Hmm I wouldn't expect that you would need to run the codegen command as part of this PR, since there were no changes to the Contentful schema or new queries/mutations. Was the run here to test that the URL was constructed correctly? Perhaps others made changes in the meantime?

@julianpoy or @xlisachan , should these codegen changes be part of the PR? Meg, are you certain that your config here is pointing to the same schema as previously? I will admit I'm not very familiar yet with this part.